File: tools_java/tools/src/tools/Results.java
package tools;
/**
 * 
 * @author broihier
 * <pre>
 * Class for creating Results objects
 * </pre>
 */
public class Results extends DatabaseComponent {
	/** 
	 * Constructor for creating Results objects
	 * 
	 * @param category name that maps to 'category'.testDbResults files
	 * 
	 */
	public Results(String category) {
		super(category+".testDbResults");
	}
	/**
	 * Method to get the results text for a particular testID
	 * @param testID test case ID
	 * @return getComponentText(testID)
	 */
	public String getResultsText(String testID) {
		return getComponentText(testID);
	}

}

File: tools_C++/src/Results.cpp
/*
 * Results.cpp
 *
 *  Created on: Apr 12, 2017
 *      Author: broihier
 */

#include "Results.h"

Results::Results(std::string category): DatabaseComponent(category+".testDbResults") {
}

Results::~Results() {
}

const std::string& Results::getResultsText(const std::string& id) const{
	return getComponentText(id);
}

